solver/src/solver/single_order_solver.rs

```diff
@@ -86,13 +86,16 @@ impl<I: SingleOrderSolving + Send + Sync + 'static> Solver for SingleOrderSolver
 pub struct SettlementError {
     pub inner: anyhow::Error,
     pub retryable: bool,
+    // Whether or not this error should be logged as an error
+    pub should_alert: bool,
```

Comment:
I see this new filed but it is never read is it? So we set it to false but we don't actually prevent the alert at the moment.

Reply:
Good point, forgot to push the latest commit